src/box/lua/console.lua

```diff
@@ -118,23 +120,35 @@ output_handlers["lua"] = function(status, opts, ...)
 end
 
 local function parse_output(value)
-    local fmt, opts
     if not value then
         return 'Specify output format: lua or yaml.'
     end
+    local fmt, opts, local_eos
     for _, v in ipairs(value:split(',')) do
         if v == 'yaml' or v == 'lua' then
             fmt = v
         elseif v == 'line' or v == 'block' then
             opts = v
+        elseif v:find('=') then
+            local pivot = v:find('=')
+            local lhs = v:sub(1, pivot - 1)
+            local rhs = v:sub(pivot + 1, -1)
+            if lhs == 'local_eos' then
+                local_eos = rhs
+            else
+                return ('Unknown "\\set output" option: %q'):format(lhs)
+            end
         else
             return ('Unknown "\\set output" option: %q'):format(v)
         end
     end
-    if opts and fmt ~= 'lua' then
+    if type(fmt) ~= 'string' then
```

Comment:
Minor: Is this check required? If it is, please move it to the second patch; otherwise, just drop it.

Reply:
We need `fmt` check, so `"Invalid language nil, opts are available only in lua"` error message would not appear. Changed it to `fmt ~= nil` (it is equivalent in this context).